src/Search/SearchHandler.php

```diff
@@ -26,9 +26,20 @@ class SearchHandler
      */
     protected $pool;
 
-    public function __construct(Pool $pool)
+    /**
+     * @var bool
+     */
+    private $caseSensitive;
+
+    /**
+     * NEXT_MAJOR: remove default true value for $caseSensitive and add bool type hint.
+     *
+     * @param bool $caseSensitive
+     */
+    public function __construct(Pool $pool, $caseSensitive = true)
```

Comment:
Do you even need default true here since you have it in bundle configuration?

Reply:
One could use it customized cause it is not final yet, so from the code POV I would say yes

Reply:
Yep, forgot it is not final